### ISSUE TITLE
LazyInitializationException while starting new batch export

### ISSUE THREAD

**Issue by henning-gerhardt** [goobi/goobi-production#445]
While further investigations of #431 I started a second newspaper batch export while first was still running. After starting second export inside taskmanager window folling error message appear on previous started batch:

```
Fehler (LazyInitializationException while exporting ArtiNo_425390144-18220117: failed to lazily initialize a collection of role: de.sub.goobi.beans.Projekt.filegroups, no session or session was closed)
```

Both batches does not include same process twice. One batch contains processes of year 1822 and second batch contains processes of year 1826.

**Comment by henning-gerhardt** [comment]
Issue can also be trigged while batch export is running, switching to batch form and switching back to task manager window.

**Comment by henning-gerhardt** [comment]
Fixed by #452
